endpoints/meeting-search.php

```diff
@@ -26,9 +26,21 @@
     $text_space = " ";
     $message = "";
 
+function mobileCheck()
+{
+    $is_mobile = true;
+    if (has_setting('mobile_check') && json_decode(setting('mobile_check'))) {
+        $phone_number = $GLOBALS['twilioClient']->lookups->v1->phoneNumbers($_REQUEST['From'])->fetch(array("type" => "carrier"));
+        if ($phone_number->carrier['type'] !== 'mobile') {
+            $is_mobile = false;
```

Comment:
I would just `return false;` here, and then `return true;` at the bottom of the function. That way you don't even need `$is_mobile`".

It's already merged, and the code works, so no need to change it - but next time :).

Reply:
ah yes that totes makes sense. I initially was using $is_mobile but after code changes its no longer needed.